app/css/interceptor.css

```diff
@@ -649,6 +648,11 @@ svg.spinner > circle {
 	display: grid;
 }
 
+.swap-grid-1 {
+	grid-template-columns: auto;
+	display: grid;
+}
+
 .textbox {
 	background-color: var(--alpha-005);
 	padding: 10px;
```

Comment:
it's out of the view of changes but there's another set of extra spaces in line :722 (2 spaces) while you're at it